wapm-toml/src/lib.rs

```diff
@@ -625,18 +658,37 @@ name = "mod"
 source = "target/wasm32-wasi/release/mod.wasm"
 interfaces = {"wasi" = "0.0.0-unstable"}
 
+[[module]]
+name = "mod-with-exports"
+source = "target/wasm32-wasi/release/mod-with-exports.wasm"
+bindings = { wit = "exports.wit", wit_bindgen = "0.0.0" }
```

Comment:
What you think of using `wit-exports` instead of just `wit` ? (I'm thinking on how we'll do it as well when it has imports)

Reply:
Okay, let's go with `wit-exports` :ok_hand: